platforms/enterprise/enterprise/build.gradle.kts

```diff
@@ -11,31 +11,35 @@ errorprone {
 }
 
 dependencies {
+    api(project(":build-operations"))
```

Comment:
I didn't check the individual dependency changes, I suppose if it works all is good 👍 .